### PR TITLE
[dynamo][graph break fix] inplace add for empty tuple

### PR DESCRIPTION
**Pull request by anijain2305**
Fixes one of the frequent graph breaks in HF models.

cc @soumith @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

**Review by yanboliang — APPROVED** [review]

**Comment by yanboliang** [comment]
If you reduced graph breaks for benchmark/huggingface, you probably have to update https://github.com/pytorch/pytorch/tree/master/benchmarks/dynamo/ci_expected_accuracy

**Review by jansel — APPROVED** [review]

File: torch/_dynamo/symbolic_convert.py [hunk 1]
```diff
@@ -675,7 +675,11 @@ def LOAD_CLOSURE(self, inst):
         self.push(ClosureVariable(name=inst.argval))
 
     def LOAD_CONST(self, inst):
-        self.push(ConstantVariable(value=inst.argval))
+        # For empty tuples, create empty TupleVariable
+        if isinstance(inst.argval, tuple) and not inst.argval:
+            self.push(TupleVariable([]))
```
Comment (jansel) [inline torch/_dynamo/symbolic_convert.py:680]: Maybe we should call into VariableBuilder to do this in a more general way...

Comment (anijain2305) [reply torch/_dynamo/symbolic_convert.py:680]: I will look into this a separate PR.

**Pull request closed without merging**